aws-analytics-pinpoint/src/main/java/com/amplifyframework/analytics/pinpoint/AWSPinpointAnalyticsPluginBehavior.kt

```diff
@@ -22,18 +22,46 @@ import com.amplifyframework.analytics.AnalyticsDoubleProperty
 import com.amplifyframework.analytics.AnalyticsEventBehavior
 import com.amplifyframework.analytics.AnalyticsIntegerProperty
 import com.amplifyframework.analytics.AnalyticsProperties
+import com.amplifyframework.analytics.AnalyticsPropertyBehavior
 import com.amplifyframework.analytics.AnalyticsStringProperty
 import com.amplifyframework.analytics.UserProfile
+import com.amplifyframework.analytics.pinpoint.models.AWSPinpointUserProfile
+import com.amplifyframework.analytics.pinpoint.targeting.TargetingClient
+import com.amplifyframework.analytics.pinpoint.targeting.endpointProfile.EndpointProfileUser
 
 internal class AWSPinpointAnalyticsPluginBehavior(
     private val context: Context,
     val analyticsClient: AnalyticsClient,
+    private val targetingClient: TargetingClient,
     private val autoEventSubmitter: AutoEventSubmitter,
     private val autoSessionTracker: AutoSessionTracker
 ) : AnalyticsCategoryBehavior {
 
+    private val userName = "name"
```

Comment:
Looks like you aren't re-susing these so i'd be fine with using these variables at all, but if we do use them, probably best as a const val in companion object.